src/main/webapp/app/exercises/shared/team/team.service.ts

```diff
@@ -201,6 +228,38 @@ export class TeamService implements ITeamService {
         return this.http.get<Course>(`${SERVER_API_URL}api/courses/${course.id}/teams/${team.shortName}/with-exercises-and-participations`, { observe: 'response' });
     }
 
+    /**
+     * Exports given teams into json file
+     * @param teams Teams to add to export file
+     */
+    exportTeams(teams: Team[]) {
+        // Make list of questions which we need to export,
+        const exportedTeams: TeamList = { students: [] };
+        const studentsWithoutRegistrationNumbers: { login: string; name: string }[] = [];
+        teams!.forEach((team) => {
+            if (team.students) {
+                team.students.forEach((student) => {
+                    const exportStudent = new StudentWithTeam();
+                    exportStudent.Name = student.firstName || '';
```

Comment:
You need to watch out when using the `||` operator if you want to use a default value when `student.firstName === undefined`. It's [better](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to use `??`